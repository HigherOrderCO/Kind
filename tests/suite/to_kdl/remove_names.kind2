List <a> : Type
  List.nil <a> : List
  List.cons <a> (head: a) (tail: List a) : List

Ora.Ora.ora (h: List U60) : List U60
Ora.Ora.ora (List.cons x xs) =
  let a = 2
  let b = 3
  let c = 4
  let d = (+ c b)
  List.cons a xs