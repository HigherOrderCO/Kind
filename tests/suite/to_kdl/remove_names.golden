// List.nil #List.nil -(a: Type) : (List _)
ctr {List.nil}

// List.cons #List.cons -(a: Type) (head: a) (tail: (List a)) : (List _)
ctr {List.cons head tail}

// Ora.Ora.ora #Ora.Ora.ora (h: (List U60)) : (List U60)
fun (Ora.Ora.ora h) {
  (Ora.Ora.ora {List.cons ~ xs}) = let a = #2; let b = #3; let c = #4; let ~ = (+ c b); {List.cons a xs}
}

